docker/.env

```diff
@@ -38,6 +38,8 @@ REACT_APP_PUBLIC_GRAPH_URI=https://localhost:8082/public
 REDIS_EVENTS_STAGING_ENDPOINT=redis:6379
 RENDER_PREVIEW=true
 SESSION_FILE_PATH_PREFIX=/tmp/
+REACT_APP_AUTH_MODE=password
```

Comment:
Intentionally setting REACT_APP_AUTH_MODE to password to facilitate testing.
I feel this ought to be the default

Reply:
sounds good about making it the default, but that may require updating the e2e test to allow the cypress instance to log in with the default password.
some other issues with this:
- there are some places this is overridden, ie. `docker/env.sh`. you'll need to update that
- the password will need to be provided to the backend container for dev and hobby modes. see the `BUILD_ARGS` value, you'll need to update `backend.Dockerfile` as well
- for hobby mode, `./start-hobby.sh` should make sure the password is set to a non-default value. the shell script should exit if it is not set

Reply:
I just updated the cypress login e2e test to login with ADMIN_PASSWORD. It succeeded when I run it locally with "chrome" option.

I updated the BUILD_ARGS to set ADMIN_PASSWORD

Also modified the start-hobby.sh to exit with appropriate error message if ADMIN_PASSWORD is not set.

Only issue is the backend.Dockerfile. I'm not sure what to do there. Can you clarify.

Reply:
cool, thanks for the updates here! regarding `backend.Dockerfile`, disregard - I thought the env var would have to be baked in to the image, but i see it's just passed as an env var via the `compose.hobby.yml` to the running backend container.